dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1236,7 +1236,7 @@ public Workflow manualRegister(@ApiParam(hidden = true) @Auth User user,
         @ApiParam(value = "Workflow registry", required = true) @QueryParam("workflowRegistry") String workflowRegistry,
         @ApiParam(value = "Workflow repository", required = true) @QueryParam("workflowPath") String workflowPath,
         @ApiParam(value = "Workflow container new descriptor path (CWL or WDL) and/or name", required = true) @QueryParam("defaultWorkflowPath") String defaultWorkflowPath,
-        @ApiParam(value = "Workflow name", required = true) @QueryParam("workflowName") String workflowName,
+        @ApiParam(value = "Workflow name, set to empty if none required", required = true) @QueryParam("workflowName") String workflowName,
```

Comment:
This seems complicated, why not just remove `required = true`?

If you do keep it this way, with `required=true`, then the null check on line 1261 is redundant, but harmless.

Reply:
@coverbeck this change is good in theory, but seems to break a bunch of testing code, so I'll delay the change to the next release

Reply:
👍